packages/sdk/src/sync-agent/syncAgents.test.ts

```diff
@@ -75,4 +75,38 @@ describe('syncAgents.test.ts', () => {
             { timeoutMS: 10000 },
         )
     })
+
+    test('syncAgents pin a message', async () => {
+        await Promise.all([bob.start(), alice.start()])
+        await waitFor(() => bob.spaces.value.status === 'loaded')
+        expect(bob.spaces.data.spaceIds.length).toBeGreaterThan(0)
+        const spaceId = bob.spaces.data.spaceIds[0]
+        expect(alice.user.memberships.isJoined(spaceId)).toBe(true) // alice joined above
+        const space = bob.spaces.getSpace(spaceId)
+        const channel = space.getDefaultChannel()
+        const channelId = channel.data.id
+        const event = channel.timeline.events.value.find((e) => e.text === 'Hello, World!')
+        expect(event).toBeDefined()
+        // bob can pin
+        const result = await channel.pin(event!.eventId)
+        expect(result).toBeDefined()
+        expect(result.error).toBeUndefined()
+        await waitFor(() =>
+            expect(
+                bob.riverConnection.client?.streams.get(channelId)?.view.membershipContent.pins
+                    .length,
+            ).toBe(1),
+        )
+        // bob can unpin
+        const result2 = await channel.unpin(event!.eventId)
+        expect(result2).toBeDefined()
+        expect(result2.error).toBeUndefined()
+
```

Comment:
wdyt about adding a wait for membership content pins length to go back to 0? (is that too pedantic?)